xdsl/dialects/riscv.py

```diff
@@ -100,10 +128,22 @@ class SubOp(RdRsRsOp):
     name = "riscv.sub"
 
 
+@irdl_op_definition
+class LiOp(RdImmOp):
```

Comment:
Can we add something here to mark it as a pseudo op?

Reply:
We can probably use traits for that?